canonical-websites/www.ubuntu.com
# Update /legal/trademarks page and converted to markdown

## Done

- Updated the list of trademarks
- Updated the [copy doc](https://docs.google.com/document/d/1o4P7HEIop3o4oibUH8rbj6mxIBmHakt-djZlpcpTW-I/edit#)
- Converted it to use Markdown

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/legal/trademarks](http://0.0.0.0:8001/legal/trademarks)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- See that the page looks like the old html one except for the changes in the copy doc

## Issue / Card

Fixes #4648
